2025/webextensions-wg.html

```diff
@@ -159,285 +206,629 @@ <h2>Scope</h2>
-          <li><strong>Web Extensions APIs</strong> - the interface extensions are built on.</li>
-          <li><strong>Resource files and formats</strong>, including a manifest format and how extensions are localized. The manifest format used by extensions predates and is notably different from the <a href="https://www.w3.org/TR/appmanifest/">Web Application Manifest</a> specification.</li>
-          <li><strong>Native Messaging</strong> - API surface extensions to exchange messages with native applications running on the host system.</li>
-          <li><strong>WebDriver integration</strong></li>
+          <li>
+            <strong>User agent extensibility</strong> - user agents may provide
+            APIs and other interaction points that allow third-party code (web
+            extensions) to deeply integrate with intermediate client features,
+            concepts, capabilities, and systems that may not be exposed to the
+            open web. These include, but are not limited to, tab management,
+            bookmarks, cookies stores, and browser settings.
+          </li>
+          <li>
+            <strong>Extensions model</strong> - the basic architecture of
+            extensions and how the different pieces interrelate. This definition
+            will include a description of the trust model, the differing trust
+            between browsers and extensions, between extensions, and between
+            extensions and web pages.
+          </li>
+          <li>
+            <strong>Permissions model</strong> - what sorts of powerful features
+            extensions may access, and how extensions may request user consent
+            to access them. The permissions model and API used by extensions
+            predates and is notably different from the
+            <a href="https://www.w3.org/TR/permissions/">Permissions</a>
+            specification.
+          </li>
+          <li>
+            <strong>Web Extensions APIs</strong> - the interfaces that
+            extensions are built on.
+          </li>
+          <li>
+            <strong>Resource files and formats</strong> - assets created by
+            third-party developers that are consumed by the user agent. Assets
+            include, but are not limited to, a manifest format and files
+            specifying how a given extension is localized. The extension
+            manifest format predates and is notably different from the
+            <a href="https://www.w3.org/TR/appmanifest/"
+              >Web Application Manifest</a
+            >
+            specification.
+          </li>
+          <li>
+            <strong>Native Messaging</strong> - API surface extensions to
+            exchange messages with native applications running on the host
+            system.
+          </li>
+          <li>
+            <strong>WebDriver integration</strong> - the production of artifacts
```

Comment:
Perhaps a touch more specific would be helpful. If "artifacts" entails test harnesses, conformance tooling, or something else, perhaps mention that?